jarvis_female/README.md

```diff
@@ -0,0 +1,11 @@
+Add these substitution to your YAML config on your S3 Box
+
+```yaml
+substitutions:
+  loading_illustration_file: https://github.com/MrHollowPS/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/loading.png
+  idle_illustration_file: https://github.com/MrHollowPS/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/idle.png
+  listening_illustration_file: https://github.com/MrHollowPS/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/listening.png
+  thinking_illustration_file: https://github.com/MrHollowPS/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/thinking.png
+  replying_illustration_file: https://github.com/MrHollowPS/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/replying.png
+  error_illustration_file: https://github.com/MrHollowPS/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/error.png
```

Comment:
```suggestion
  loading_illustration_file: https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/loading.png
  idle_illustration_file: https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/idle.png
  listening_illustration_file: https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/listening.png
  thinking_illustration_file: https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/thinking.png
  replying_illustration_file: https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/replying.png
  error_illustration_file: https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/raw/main/jarvis_female/illustrations/error.png
```

So that the URL points to this repo ;)